modules/api/api-documentation/api.yaml

```diff
@@ -71,6 +71,16 @@ paths:
           required: false
           schema: 
             type: string
+        - name: start
+          in: query
+          required: false
+          schema: 
+            type: integer
+        - name: count
+          in: query
+          required: false
+          schema: 
+            type: integer
```

Comment:
Please can we update the response type as well

Reply:
Updated.